api/graphql/schema/mutations.graphql

```diff
@@ -42,6 +42,28 @@ type AddCommentPayload {
     operation: AddCommentOperation!
 }
 
+input EditCommentInput {
+    """A unique identifier for the client performing the mutation."""
+    clientMutationId: String
+    """"The name of the repository. If not set, the default repository is used."""
+    repoRef: String
+    """The bug ID's prefix."""
+    prefix: String!
+    """The new message to be set."""
```

Comment:
This here is missing the `target` field for `EditCommentRaw` in `api/graphql/resolvers/mutation.go`

Reply:
Setting target type to String will result in following error:
`cannot use input.Target (type string) as type entity.Id in argument to b.EditCommentRaw`
But using entity.Id as Type won't work either as entity.Id is unkown on generation and importing via the go import directive won't work obviously.
How to resolve this? 
@MichaelMure do you have an idea? Maybe I'm on a complete wrong track :disappointed:

Reply:
huhu, that happens.

Piggy-backing on that quite useless comment as I don't have another way to contact you, did you receive my email?

Reply:
Oh sorry, I look rather rarely into my mail. Will take a look.